scout/adapter/mongo/disease_terms.py

```diff
@@ -15,7 +15,13 @@
 class DiagnosisHandler(object):
     """Class for handling OMIM and disease-related database objects"""
 
-    def query_disease(self, query: str = None, source: str = None, limit: int = None) -> Iterable:
+    def query_disease(
```

Comment:
Nah doesn't look like that api function is affected. Tested on stage:

<img width="1250" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/08a7a375-d72e-4ed3-bd05-19b352e577f8">

On prod it looks the same:

<img width="1270" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/47868d4e-6d91-4f35-a96b-5a6c469a8603">

Reply:
My intention was that it should have the same behaviour since the DISEASE_FILTER_PROJECT is applied if a filter is not used in the function call. So good it seems to be working as intended.